Formula/sslmate.rb

```diff
@@ -5,8 +5,9 @@ class Sslmate < Formula
 
   desc "Buy SSL certs from the command-line"
   homepage "https://sslmate.com"
-  url "https://packages.sslmate.com/other/sslmate-1.8.0.tar.gz"
-  sha256 "0354e41a2e9bd69b167e5374a20335affa584955dc3a2e9a4046fac6d3ac02c9"
+  url "https://packages.sslmate.com/other/sslmate-1.9.0.tar.gz"
+  sha256 "3e40122484491f59178de80e14ccf7e90cea4fea94056b25c7f89abe31685b98"
+  license "MIT"
```

Comment:
Do you have a reference for the license?

Reply:
yes

Reply:
attach it in the comment.